policyengine_us/variables/gov/states/hi/tax/income/credits/hi_food_excise_credit.py

```diff
@@ -0,0 +1,51 @@
+from policyengine_us.model_api import *
+
+
+class hi_food_excise_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii food and excise tax credit"
+    defined_for = StateCode.HI
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.capitol.hawaii.gov/hrscurrent/Vol04_Ch0201-0257/HRS0235/HRS_0235-0055_0085.htm"
+
+    def formula(tax_unit, period, parameters):
```

Comment:
the person cannot be claimed as a dependent on another return, we use the "dsi" variable -> [example](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/variables/gov/states/nm/tax/income/deductions/certain_dependents/nm_deduction_for_certain_dependents_eligible.py)